### PR TITLE
ported piratebay to bs4

### PR DESCRIPTION
**Pull request by k4u5h1k**
Ported piratebay from regex to beautifulsoup and added direct link to torrents.
Screenshot 2021-04-11 at 12 52 49 AM

### CODE REVIEW

File: modules/search/piratebay.py [hunk 1]
```diff
@@ -46,10 +46,11 @@ def module_run(self):
         for item in output:
                 print()
                 self.output(item['title'])
+                self.output(item['link'])
                 self.output(item['magnet'])
-                self.output(item['date']+' '+item['uploader'])
-                self.output('Seeders  :'+item['seeders'])
-                self.output('Leechers :'+item['leechers'])
+                self.output(item['dateuploader'])
+                self.output('Seeders  : '+item['seeders'])
```
Comment (saeeddhqan) [inline modules/search/piratebay.py]: use f"". next line as well

File: core/util/piratebay.py [hunk 2]
```diff
@@ -16,19 +16,16 @@
 """
 
 import re
```
Comment (saeeddhqan) [inline core/util/piratebay.py]: re and html, still need?

**Comment by k4u5h1k** [comment]
sorry had re for testing, removed that
html is needed for unescaping &amp &nbsp and stuff

**Comment by saeeddhqan** [comment]
I think you need to use sublime text regarding tab error : )
Everything is alright? shall we?

**Comment by k4u5h1k** [comment]
Ah I hadnt fixed my tab back when I first wrote the module I guess. Its all good now.

**Pull request merged**